test/pom.xml

```diff
@@ -36,6 +36,10 @@
             <artifactId>axon-eventsourcing</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
```

Comment:
Shouldn't this be test scope or optional?

Reply:
Yes, fixed, and checked all other dependencies for this.